src/resources/postcss/datatables.pcss

```diff
@@ -0,0 +1,435 @@
+/*
+ * Table styles
+ */
+table.dataTable {
+  border-collapse: separate;
+  border-spacing: 0;
+  clear: both;
+  margin: 0 auto;
+  width: 100%;
+
+	thead {
+		th:active,
+		td:active {
+			outline: none;
+		}
+
+		.sorting,
+		.sorting_asc,
+		.sorting_desc {
+			cursor: pointer;
+			*cursor: hand;
+		}
+
+		.sorting,
+		.sorting_asc,
+		.sorting_desc,
+		.sorting_asc_disabled,
+		.sorting_desc_disabled {
+			background-position: center right;
+			background-repeat: no-repeat;
+		}
+
+		.sorting {
+			background-image: url("../../../vendor/datatables/media/images/sort_both.png");
+		}
+		.sorting_asc {
+			background-image: url("../../../vendor/datatables/media/images/sort_asc.png");
+		}
+		.sorting_desc {
+			background-image: url("../../../vendor/datatables/media/images/sort_desc.png");
+		}
+		.sorting_asc_disabled {
+			background-image: url("../../../vendor/datatables/media/images/sort_asc_disabled.png");
+		}
+		.sorting_desc_disabled {
+			background-image: url("../../../vendor/datatables/media/images/sort_desc_disabled.png");
+		}
+	}
+
+	&.widefat {
+		thead,
+		tfoot {
+			th,
+			td {
+				input {
+					margin: 0 0 0 8px;
+					vertical-align: text-top;
+				}
+			}
+
+			td.check-column {
+				padding-top: 4px;
+				vertical-align: middle;
+			}
+		}
+
+		thead,
+		tfoot,
+		tbody {
+			th.check-column {
+				padding: 11px 0 0 3px;
+			}
+		}
+	}
+
+	.check-column {
+		width: 2.2em;
+	}
+
+	&.row-border tbody th, &.row-border tbody td, &.display tbody th, &.display tbody td {
+		border-top: 1px solid #ddd;
+	}
```

Comment:
Done :) I didn't clean up the copied css file as much as I should have. Good catch.